### PR TITLE
Bump aurora-postgresql engine version and disable auto_minor_version_upgrade

### PR DESCRIPTION
**Pull request by tonejito**
* Update `variable.local_ip` URL
    + Replaced `ipv4.icanhazip.com` with `checkip.amazonaws.com`

* Bump **aurora-postgresql** `engine_version` to `10.7`
    + Also created variable to hold `engine_version` for **aurora-postgresql**

* Set **aurora-postgresql** `auto_minor_version_upgrade` to `false`
    + Created variable to allow export and override

Relates to:

* <https://support.astronomer.io/hc/en-us/requests/1281>
* <https://forum.astronomer.io/t/disable-auto-minor-version-upgrade-for-aurora-rds-cluster/447>

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -153,6 +153,18 @@ variable "db_replica_count" {
 
 variable "local_ip" {
   description = "URL used to find user's local IP for use with bastion host"
-  default     = "http://ipv4.icanhazip.com"
+  default     = "http://checkip.amazonaws.com"
```
Comment (ianstanton) [inline variables.tf:156]: What's the reason for this change?

File: variables.tf [hunk 2]
```diff
@@ -153,6 +153,18 @@ variable "db_replica_count" {
...
 }
+
+variable "engine_version" {
+  description = "Aurora database engine version."
+  type        = string
+  default     = "10.7"
+}
+
+variable "auto_minor_version_upgrade" {
+  description = "Determines whether minor engine upgrades will be performed automatically in the maintenance window"
```
Comment (ianstanton) [inline variables.tf]: Looks good! Can you add something like "...for aurora db" at the end so we know which resource this is for?

File: variables.tf (hunk 1 above)
Comment (tonejito) [reply variables.tf:156]: On december and early january the `icanhazip.com` domain was having issues (can't remember if the domain wasn't renewed or the web host wasn't responding), that caused noise when running the terraform on our side.

I haven't had issues with checkip.amazonaws.com ever. Another alternative would be checkip.dns.he.net but that will return your IPv6 address, which could be a gotcha since we expect an IPv4 address returned by this.

File: variables.tf [hunk 3]
```diff
@@ -153,6 +153,18 @@ variable "db_replica_count" {
...
   description = "URL used to find user's local IP for use with bastion host"
-  default     = "http://ipv4.icanhazip.com"
+  default     = "http://checkip.amazonaws.com"
   type        = string
 }
+
+variable "engine_version" {
+  description = "Aurora database engine version."
+  type        = string
+  default     = "10.7"
```
Comment (ianstanton) [inline variables.tf:163]: @tonejito and @sjmiller609 What do you think about bumping this to the latest `11.6`? https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraPostgreSQL.Updates.20180305.html

**Review by ianstanton — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 2 above)
Comment (tonejito) [reply variables.tf]: Sure, I'll modify it.

File: variables.tf (hunk 3 above)
Comment (tonejito) [reply variables.tf:163]: I bumped it to `10.7` because our original instance was `10.6` (as the module specified) and it got updated to `10.7` on a maintenance window because `auto_minor_version_upgrade` was set to `true`

Don't know if hardcoding another version would force our Aurora RDS db to **create** a new instance or **migrate** the existing one :S

File: variables.tf (hunk 1 above)
Comment (ianstanton) [reply variables.tf:156]: Makes sense!

File: variables.tf (hunk 3 above)
Comment (sjmiller609) [reply variables.tf:163]: I'm OK with 10.7 here

Comment (ianstanton) [reply variables.tf:163]: I had 11.6 in my CI run and it failed. Let's stick with 10.7 for now and consider updating this in the future.

**Review by sjmiller609 — APPROVED** [review]
LGTM, just make sure CI linting passes

**Comment by ianstanton** [comment]
Looks like CI is failing on lint. @tonejito Can you run `terraform fmt` and push again?

**Comment by tonejito** [comment]
Sure, I'll run `terraform fmt` and push again

**Comment by ianstanton** [comment]
@tonejito Were you able to make the changes and run `terraform fmt`?

**Comment by tonejito** [comment]
@ianstanton I just pushed the `terraform fmt` changes, waiting for CI

File: variables.tf (hunk 2 above)
Comment (tonejito) [reply variables.tf]: Added comment on `e6ce1af`

**Review by ianstanton — APPROVED** [review]

**Review by sjmiller609 — APPROVED** [review]
